tests/unit/test_routes.py

```diff
@@ -446,13 +437,65 @@ def test_create_resource(module_client, module_db, fake_auth_from_oc, fake_algol
 
     # Invalid Resource Path
     response = client.post('/api/v1/resources',
-        headers = {'x-apikey': apikey}
-    )
+                           headers={'x-apikey': apikey})
+    assert (response.status_code == 422)
+
+    # Bogus Data
+    name = False
+    url = "htt://bad_url.doesnotexist"
+    category = True
+    languages = False
+    paid = "Bad Data"
+    notes = True
+    response = create_resource(client,
+                               apikey,
+                               name,
+                               url,
+                               category,
+                               languages,
+                               paid,
+                               notes)
+    assert (response.status_code == 422)
+
+    # A String to Big for the DB
+    long_string = "x" * 6501
+    name = long_string
+    url = long_string
+    category = long_string
+    languages = long_string
+    paid = True
+    notes = long_string
+    response = create_resource(client,
+                               apikey,
+                               name,
+                               url,
+                               category,
+                               languages,
+                               paid,
+                               notes)
     assert (response.status_code == 422)
+
+    # Unicode Characters
+    name = "😀"
+    url = None
+    category = None
+    languages = None
+    paid = True
+    notes = "∞"
+    response = create_resource(client,
+                               apikey,
+                               name,
+                               url,
+                               category,
+                               languages,
+                               paid,
+                               notes)
+    assert (response.status_code == 200)
```

Comment:
Does this pass? That's odd. `url` is required so this should fail with a response letting the user know that `url` is required to create a resource 🤔

Reply:
This is why it didn't fail https://github.com/OperationCode/resources_api/blob/d667ac0df19a40f0599e34559708fcf56722716c/tests/unit/test_routes.py#L1004

I didn't realize it had a default category in `create_resource`